ambry-store/src/main/java/com/github/ambry/store/IndexSegment.java

```diff
@@ -571,8 +571,8 @@ void addEntry(IndexEntry entry, Offset fileEndOffset) throws StoreException {
       long operationTimeInMs = entry.getValue().getOperationTimeInMs();
       if (operationTimeInMs == Utils.Infinite_Time) {
         lastModifiedTimeSec.set(time.seconds());
-      } else if ((operationTimeInMs / Time.MsPerSec) > lastModifiedTimeSec.get()) {
-        lastModifiedTimeSec.set(operationTimeInMs / Time.MsPerSec);
+      } else if ((operationTimeInMs / TimeUnit.SECONDS.toMicros(1)) > lastModifiedTimeSec.get()) {
```

Comment:
Instead of a division, we can do, `TimeUnit.MILLISECONDS.toSeconds(operationTimeInMs)`. Same comment for other files where division is used to go from a smaller to larger unit.

Reply:
@cgtz thanks for comment. I've made few changes.
But I'm bit confused that which one is micro, and which one is milli...is there some documentation about this_/

Reply:
Generally in our codebase, whenever there is an abbreviation `ms`, it stands for milliseconds (1/1000 of a second). Typically microseconds (1/1000000 seconds) is abbreviated as `us` or `μs`. I don't believe there are many (or any) places in the ambry project that use microseconds as opposed to nanoseconds or milliseconds.